aio/content/examples/http/src/app/heroes/heroes.component.css

```
@@ -1,73 +1,66 @@
 /* HeroesComponent's private CSS styles */
+.search input {
+  width: 100%;
+  padding: .75rem;
+  margin: 1rem 0;
+  box-sizing: border-box;
+}
 .heroes {
   margin: 0 0 2em 0;
   list-style-type: none;
   padding: 0;
-  width: 15em;
+  width: 100%;
 }
 .heroes li {
   position: relative;
   cursor: pointer;
-  background-color: #EEE;
-  margin: .5em;
-  padding: .3em 0;
-  height: 1.6em;
-  border-radius: 4px;
-  width: 19em;
 }
 
 .heroes li:hover {
-  color: #607D8B;
-  background-color: #DDD;
   left: .1em;
 }
 
 .heroes a {
-  color: #888;
+  color: black;
   text-decoration: none;
-  position: relative;
   display: block;
-  width: 250px;
+  font-size: 1.2rem;
+  background-color: #EEE;
```

Nit (for consistency):

```suggestion
  background-color: #eee;
```